Description: Prevent makefile from deleting debian/
Author: Harlan Lieberman-Berg <hlieberman@debian.org>
Forwarded: not-neededIndex: ansible/Makefile
===================================================================
--- a/Makefile
+++ b/Makefile
@@ -154,9 +154,6 @@
 	rm -f test/units/.coverage*
 	rm -rf test/results/*/*
 	find test/ -type f -name '*.retry' -delete
-	@echo "Cleaning up Debian building stuff"
-	rm -rf debian
-	rm -rf deb-build
 	rm -rf docs/json
 	rm -rf docs/js
 	@echo "Cleaning up docsite"
